src/App.js

```diff
@@ -11,11 +11,14 @@ import StartPage from './Pages/StartPage/StartPage'
 import GuestPage from './Pages/GuestPage/GuestPage'
 import PacklistForm from './Pages/PacklistForm/PacklistForm'
 
+
 export default function App() {
+
   const user = useContext(UserContext);
-  console.log('app user', user, !!user);
+  console.log('user', user)
   return (
-    <UserProvider>
+    <>
+      <GlobalStyles userLoggedIn={user ? true : false} />
```

Comment:
As an alternative, you can convert `user` to a boolean like this:

```suggestion
      <GlobalStyles userLoggedIn={Boolean(user)} />
```

or

```suggestion
      <GlobalStyles userLoggedIn={!!user} />
```